queries/eth_spent.sql

```diff
@@ -0,0 +1,21 @@
+-- V1: https://dune.com/queries/1320169
+select concat('0x', encode(solver_address, 'hex')) as receiver,
+       sum(gas_price_gwei * gas_used) * 10 ^ 9     as amount
+from gnosis_protocol_v2.batches
+join gnosis_protocol_v2.view_solvers
+    on solver_address = address
+where block_time between '{{StartTime}}' and '{{EndTime}}'
+  and environment not in ('services', 'test')
+group by solver
+order by eth_spent desc
```

Comment:
I get an `Error: Column "eth_spent" does not exist at line 9, position 10.` error when trying to run this query in Dune.

Reply:
Yes! Sorry this is meant to be receiver. I thought I had already fixed it... will check again.